app/controllers/ontologies_controller.rb

```diff
@@ -38,6 +38,7 @@ def index
     @groups = LinkedData::Client::Models::Group.all(display_links: false, display_context: false)
 
     @filters = ontology_filters_init(@categories, @groups)
+    @federation_enabled = federation_enabled?
```

Comment:
no need to put it here, directly call the helper in the view

Reply:
Done in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/780/commits/ab9f5b4e1d42adc165bbf5a92bf9dd49e102bf35